packages/server/src/utils/request-logger.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Module inspired by 'koa-logger'. Adapted and extended to our needs.
+ */
+import counterFunc from 'passthrough-counter'
+import bytes from 'bytes'
+import chalk from 'chalk'
+
+export function logRequests({ ignoreUrls } = {}) {
+  return async function middleware(ctx, next) {
+    if (ignoreUrls && ctx.req.url.match(ignoreUrls)) {
+      return next()
+    }
+    // request
+    const start = Date.now()
+
+    logRequest(ctx)
+
+    try {
+      await next()
+    } catch (err) {
+      logResponse({
+        ctx,
+        start,
+        err
+      })
```

Comment:
`logResponse({ ctx, start, err })` is nicer here I think, and prettier would even allow it.